Makefile

```diff
@@ -1,8 +1,13 @@
 SHELL=/bin/bash
 
+constants: hca/api_spec.json
+
 lint:
 	./setup.py flake8
 
+hca/api_spec.json:
+	wget https://hca-dss.czi.technology/v1/swagger.json
```

Comment:
Suggest rewriting this to `curl https://hca-dss.czi.technology/v1/swagger.json > hca/api_spec.json`